src/main/java/com/powsybl/openloadflow/network/impl/LfTopoConfig.java

```diff
@@ -22,6 +22,10 @@ public class LfTopoConfig {
 
     private final Set<String> busIdsToLose = new HashSet<>();
 
+    private final Set<String> branchIdsWithPtcToRetain = new HashSet<>();
```

Comment:
For later, but maybe we should find a better name for LfTopoConfig once this will be added?

Reply:
We can do it now, shunt compensator action is coming just after, what do you suggest ?

Reply:
Could be a `LfNetworkLoadingParameters`

Reply:
I think we can leave it like this and re-think about it with shunt compensator action.